firmware/boron-ins-fsm/src/stateMachine.cpp

```diff
@@ -260,26 +266,29 @@ void state3_stillness(){
     Log.warn("door opened, session over, going from state3_stillness to idle");
     publishStateTransition(3, 0, checkDoor.doorStatus, checkINS.iAverage);
     saveStateChange(3, 2);
+    hasDurationAlertBeenSent = 0; // session is over, so reset flag
```

Comment:
What do you think about resetting `hasDurationAlertBeenSent` in `state0_idle` instead of in both state 2 and state 3?

Reply:
I like this more, thanks. I made the change in my latest commit